caffeine@patapon.info/extension.js

```diff
@@ -147,7 +148,10 @@ const CaffeineToggle = GObject.registerClass(
 class CaffeineToggle extends QuickSettings.QuickMenuToggle {
     _init() {
         super._init({
-            label: IndicatorName,
+            // The 'label' property was renamed to 'title' in GNOME 44 but quick settings have otherwise 
+            // not been changed. The below line allows support for both GNOME 43 and 44+ by using the 
+            // appropriate property name based on the GNOME version.
+            [Number(Config.PACKAGE_VERSION.split('.')[0]) >= 44 ? 'title' : 'label']: IndicatorName,
```

Comment:
Now there is also a `subtitle` property. This could be used to display the countdown timer. ;)

Reply:
Also, thanks for updating that to 44 so quickly! :D